### PR TITLE
:partysloth:

### PR DESCRIPTION
**Pull request by jenlersadnie**
Sloths are nice, and I hear they like to party.

### CODE REVIEW

File: parrots.json [hunk 1]
```diff
@@ -576,5 +576,9 @@
       "hd": "hd/redenvelopeparrot.gif",
       "name": "Red Envelope Parrot",
       "tip": "Happy Lunar New Year!"
+    },
+    {
+      "hd": "hd/partysloth.gif",
```
Comment (jmhobbs) [inline parrots.json]: Looks like there is a name mismatch from `sloth.gif` to `partysloth.gif`.

Comment (jenlersadnie) [reply parrots.json]: Oh, d'oh! Good catch. Too much partysloth excitement to think clearly on this Friday evening.

**Comment by jenlersadnie** [comment]
I've updated my fork with both of those requested changes. Let me know what you think!

**Comment by jmhobbs** [comment]
Perfect! Thanks!

**Pull request merged**